nncf/openvino/quantization/quantize_model.py

```diff
@@ -139,7 +138,7 @@ def native_quantize_with_accuracy_control_impl(
     model: ov.Model,
     calibration_dataset: Dataset,
     validation_dataset: Dataset,
-    validation_fn: Callable[[Any, Iterable[Any]], float],
+    validation_fn: Callable[[Any, Iterable[Any]], Tuple[float, Union[None, List[float], List[List[TTensor]]]]],
```

Comment:
It looks like an adaptation of the POT implementation is required.

Reply:
Done